conans/client/userio.py

```diff
@@ -42,13 +42,10 @@ def request_login(self, remote_name, username=None):
         :param username If username is specified it only request password"""
         user_input = ''
         while not username:
-            try:
-                if self._interactive:
-                    self.out.write("Remote '%s' username: " % remote_name)
-                user_input = self.get_username(remote_name)
-                username = Username(user_input)
-            except InvalidNameException:
-                self.out.error('%s is not a valid username' % user_input)
+            if self._interactive:
+                self.out.write("Remote '%s' username: " % remote_name)
+            user_input = self.get_username(remote_name)
+            username = Username(user_input)
```

Comment:
I am not sure about this. The implementation is:

```python
class Username(str):

    def __new__(cls, name):
        """Simple name creation.

        @param name:        string containing the desired name
        """
        ConanName.validate_user(name)
        return str.__new__(cls, name)
```
So it will still raise if ``validate_user()`` pattern is not fullfilled. Sounds a test would help to make sure it is working as expected.

Reply:
You're right -- I got a different err msg after this change and assumed it was sent over from the server. On further inspection it's just displaying the exception message from models/ref.py.